types/RanvierWebSocket.d.ts

```diff
@@ -0,0 +1,27 @@
+import { EventEmitter } from 'events';
+import WebSocket from 'ws';
+
+export declare class RanvierWebSocket extends EventEmitter {
```

Comment:
I feel like some of these props also exist on telnet sockets

Reply:
I did end up moving the telnet-networking bundle into TS so I don't have this declaration class in mine. I have type from the TelnetSocket class itself.

I'm not super clear on the utility of having the streams and having sockets on the streams whilst nearly having all the same properties anyway. And since they are nearly interchangeable from the highlevel of just using socket.write...

I'm not sure exactly what the best solution is here.

Reply:
Honestly your PR here is probably fine since it improves on the existing types. I'll test it out soon and report back.